testing/ember-app/tests/utils/keep-latest/js-test.ts

```diff
@@ -42,7 +42,7 @@ module('Utils | keepLatest | js', function (hooks) {
     assert.strictEqual(instance.data, 1);
     await timeout(15);
     assert.strictEqual(instance.data, 1);
-    await timeout(40);
+    await timeout(100);
```

Comment:
So the magic number 40 failed at this point and I had to raise it to 100.

It's a shame we wait for magic number of milliseconds, but from the top of my head I can't think of a better solution.

`ember-lifeline` might solve this, but it increases code complexity enormously. 👎

Reply:
yeah :(